addons/crm/models/crm_lead.py

```diff
@@ -1269,15 +1269,18 @@ def get_empty_list_help(self, help_message):
     # BUSINESS
     # ------------------------------------------------------------
 
-    def log_meeting(self, meeting_subject, meeting_date, duration):
-        if not duration:
+    def log_meeting(self, meeting):
+        """ Log the meeting info with a link to it in the chatter
+        :param record meeting: the meeting we want to log
+        """
+        if not meeting.duration:
             duration = _('unknown')
         else:
-            duration = self.env['ir.qweb.field.duration'].value_to_html(duration, {'unit': 'hour'})
-        meet_date = fields.Datetime.from_string(meeting_date)
+            duration = self.env['ir.qweb.field.duration'].value_to_html(meeting.duration, {'unit': 'hour'})
+        meet_date = fields.Datetime.from_string(meeting.start)
         meeting_usertime = fields.Datetime.to_string(fields.Datetime.context_timestamp(self, meet_date))
-        html_time = "<time datetime='%s+00:00'>%s</time>" % (meeting_date, meeting_usertime)
-        message = _("Meeting scheduled at '%s'<br> Subject: %s <br> Duration: %s") % (html_time, meeting_subject, duration)
+        html_time = "<time datetime='%s+00:00'>%s</time>" % (meeting.start, meeting_usertime)
+        message = _("Meeting scheduled at '%s'<br> Subject: %s <br> Duration: %s") % (html_time, meeting._get_html_link(), duration)
```

Comment:
While modifying it, I would markupize and use named parameters to ease translation and move to a 2023 writing, something like
```
message = '<p>%s</p>' % Markup(
    _('Meeting scheduled at %(html_time)s<br />Subject: %(subject)s<br />Duration: %(duration)s'
) % {
    'html_time': Markup('<time datetime='%(meeting_date)s+00:00'>%(meeting_user_time)s</time>') % { 
        'meeting_date': meeting_date,
        'meeting_usertime': meeting_usertime,
    },
    'subject': meeting._get_html_link(),  # already Markup valid
    'duration': duration,
}
```